mappings/net/minecraft/packet/play/SendChatMessageC2S.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_340 net/minecraft/packet/play/SendChatMessageC2S
```

Comment:
can this just be send chat or send messge? chat and message have same meanings.

Reply:
I disagree. `SendChat` isn't clear if you're sending one message or if you're sending in batch, and `SendMessage` isn't clear on what kind of message